pac4j-core/src/main/java/org/pac4j/core/exception/MultipleAccountsFoundException.java

```diff
@@ -6,7 +6,7 @@
  * @author Jerome Leleu
  * @since 1.8.0
  */
-public class MultipleAccountsFoundException extends TechnicalException {
+public class MultipleAccountsFoundException extends CredentialsException {
```

Comment:
OK. I see. It's certainly more an abnormal problem than an expected one. Will revert to a `TechnicalException`.